outline/intro.md

```diff
@@ -195,7 +195,7 @@ Introduction to Programming with Clojure
 
 **R**ead, **E**val, **P**rint, **L**oop
 
-![Instarepl](img/instarepl.png)
+![Nightcode's repl](img/repl.png)
```

Comment:
This is a good point. I think on LightTable, just type "Hello World!" didn't make sense well. That's why it used `prn-str`. On Nightcode, simple possible string only would work. I'll take in before push the change to repository.